shiksha-migration/cohort-academicYear-userdb-migration.js

```
@@ -0,0 +1,187 @@
+     * CONNECT DESTINATION DB
+     */
+    await destClient.connect();
+
+    console.log(
+      '[COHORT_ACADEMIC_YEAR] Connected to destination database'
+    );
+
+    const tenantId = 'ef99949b-7f3a-4a5f-806a-e67e683e38f3';
+    const adminUserId = '8058c0e1-8c78-4483-8add-495bbe774823';
+    const apiBaseUrl = 'http://localhost:3000';
+    const bearerToken = 'eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJPc3NtSUhXaW1NMDN2MUxsVnFvNHBqaS0ydEMwTGhLY0o5dmtwQTlJZV9zIn0.eyJleHAiOjE3Nzk2MzQxNjAsImlhdCI6MTc3OTU0Nzc2MCwianRpIjoiMDhmYWQ2OTEtMWIzMC00MGE5LTliMjItMjQwMTY5NDQ1MTFhIiwiaXNzIjoiaHR0cHM6Ly9kZXYtbG1wLnByYXRoYW1kaWdpdGFsLm9yZy9hdXRoL3JlYWxtcy9wcmF0aGFtIiwiYXVkIjoiYWNjb3VudCIsInN1YiI6IjgwNThjMGUxLThjNzgtNDQ4My04YWRkLTQ5NWJiZTc3NDgyMyIsInR5cCI6IkJlYXJlciIsImF6cCI6InByYXRoYW0iLCJzZXNzaW9uX3N0YXRlIjoiMzk4ODMzMWYtNjJkOC00MDVkLWI2ZjYtNmVhMzkxYzk2NjQ1IiwiYWNyIjoiMSIsImFsbG93ZWQtb3JpZ2lucyI6WyIvKiJdLCJyZWFsbV9hY2Nlc3MiOnsicm9sZXMiOlsib2ZmbGluZV9hY2Nlc3MiLCJ1bWFfYXV0aG9yaXphdGlvbiIsImRlZmF1bHQtcm9sZXMtcHJhdGhhbSJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoiZW1haWwgcHJvZmlsZSBwcmF0aGFtLXJvbGUiLCJzaWQiOiIzOTg4MzMxZi02MmQ4LTQwNWQtYjZmNi02ZWEzOTFjOTY2NDUiLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsIm5hbWUiOiJ0ZXN0IGthZmthIDYiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJzYW1wbGVrYWZrYXRlc3QzMSIsInVzZXJfcm9sZXMiOiJTdXBlciBBZG1pbiIsImdpdmVuX25hbWUiOiJ0ZXN0IiwiZmFtaWx5X25hbWUiOiJrYWZrYSA2In0.xgvo1rnQ28cJkfc9_8c5eiN5nc1NhaEInnK80IZGRN14z62uPrfvTU3elb93xK9MRh9n2HBf4jQvnwgS8E0l6bMOllXXAyg5eHQFaAgH-t9ayvfj1stxFIDvUENB9bWSH8aq7GFmhYtViUK30iT5PJx8lZ17tw0BRmaA4yXnVsANNe2ezMGNd5vJLnYIhntF0AQAp8xgu39o059eGPVl2KITrulEdm4qs5lJVZ46qO_KvQql6_fHJ_Unqisoca4c_gofK5go8pnyJhDI6z7633LY0swBFWxWmOj-xwYpghDwwkw68_16uXpeyAVoVKMcX0jXrmtAkLm8EQnqgWQpmg'
```

ESLint

[error] Replace `·'eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJPc3NtSUhXaW1NMDN2MUxsVnFvNHBqaS0ydEMwTGhLY0o5dmtwQTlJZV9zIn0.eyJleHAiOjE3Nzk2MzQxNjAsImlhdCI6MTc3OTU0Nzc2MCwianRpIjoiMDhmYWQ2OTEtMWIzMC00MGE5LTliMjItMjQwMTY5NDQ1MTFhIiwiaXNzIjoiaHR0cHM6Ly9kZXYtbG1wLnByYXRoYW1kaWdpdGFsLm9yZy9hdXRoL3JlYWxtcy9wcmF0aGFtIiwiYXVkIjoiYWNjb3VudCIsInN1YiI6IjgwNThjMGUxLThjNzgtNDQ4My04YWRkLTQ5NWJiZTc3NDgyMyIsInR5cCI6IkJlYXJlciIsImF6cCI6InByYXRoYW0iLCJzZXNzaW9uX3N0YXRlIjoiMzk4ODMzMWYtNjJkOC00MDVkLWI2ZjYtNmVhMzkxYzk2NjQ1IiwiYWNyIjoiMSIsImFsbG93ZWQtb3JpZ2lucyI6WyIvKiJdLCJyZWFsbV9hY2Nlc3MiOnsicm9sZXMiOlsib2ZmbGluZV9hY2Nlc3MiLCJ1bWFfYXV0aG9yaXphdGlvbiIsImRlZmF1bHQtcm9sZXMtcHJhdGhhbSJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoiZW1haWwgcHJvZmlsZSBwcmF0aGFtLXJvbGUiLCJzaWQiOiIzOTg4MzMxZi02MmQ4LTQwNWQtYjZmNi02ZWEzOTFjOTY2NDUiLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsIm5hbWUiOiJ0ZXN0IGthZmthIDYiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJzYW1wbGVrYWZrYXRlc3QzMSIsInVzZXJfcm9sZXMiOiJTdXBlciBBZG1pbiIsImdpdmVuX25hbWUiOiJ0ZXN0IiwiZmFtaWx5X25hbWUiOiJrYWZrYSA2In0.xgvo1rnQ28cJkfc9_8c5eiN5nc1NhaEInnK80IZGRN14z62uPrfvTU3elb93xK9MRh9n2HBf4jQvnwgS8E0l6bMOllXXAyg5eHQFaAgH-t9ayvfj1stxFIDvUENB9bWSH8aq7GFmhYtViUK30iT5PJx8lZ17tw0BRmaA4yXnVsANNe2ezMGNd5vJLnYIhntF0AQAp8xgu39o059eGPVl2KITrulEdm4qs5lJVZ46qO_KvQql6_fHJ_Unqisoca4c_gofK5go8pnyJhDI6z7633LY0swBFWxWmOj-xwYpghDwwkw68_16uXpeyAVoVKMcX0jXrmtAkLm8EQnqgWQpmg'` with `⏎······'eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJPc3NtSUhXaW1NMDN2MUxsVnFvNHBqaS0ydEMwTGhLY0o5dmtwQTlJZV9zIn0.eyJleHAiOjE3Nzk2MzQxNjAsImlhdCI6MTc3OTU0Nzc2MCwianRpIjoiMDhmYWQ2OTEtMWIzMC00MGE5LTliMjItMjQwMTY5NDQ1MTFhIiwiaXNzIjoiaHR0cHM6Ly9kZXYtbG1wLnByYXRoYW1kaWdpdGFsLm9yZy9hdXRoL3JlYWxtcy9wcmF0aGFtIiwiYXVkIjoiYWNjb3VudCIsInN1YiI6IjgwNThjMGUxLThjNzgtNDQ4My04YWRkLTQ5NWJiZTc3NDgyMyIsInR5cCI6IkJlYXJlciIsImF6cCI6InByYXRoYW0iLCJzZXNzaW9uX3N0YXRlIjoiMzk4ODMzMWYtNjJkOC00MDVkLWI2ZjYtNmVhMzkxYzk2NjQ1IiwiYWNyIjoiMSIsImFsbG93ZWQtb3JpZ2lucyI6WyIvKiJdLCJyZWFsbV9hY2Nlc3MiOnsicm9sZXMiOlsib2ZmbGluZV9hY2Nlc3MiLCJ1bWFfYXV0aG9yaXphdGlvbiIsImRlZmF1bHQtcm9sZXMtcHJhdGhhbSJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoiZW1haWwgcHJvZmlsZSBwcmF0aGFtLXJvbGUiLCJzaWQiOiIzOTg4MzMxZi02MmQ4LTQwNWQtYjZmNi02ZWEzOTFjOTY2NDUiLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsIm5hbWUiOiJ0ZXN0IGthZmthIDYiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJzYW1wbGVrYWZrYXRlc3QzMSIsInVzZXJfcm9sZXMiOiJTdXBlciBBZG1pbiIsImdpdmVuX25hbWUiOiJ0ZXN0IiwiZmFtaWx5X25hbWUiOiJrYWZrYSA2In0.xgvo1rnQ28cJkfc9_8c5eiN5nc1NhaEInnK80IZGRN14z62uPrfvTU3elb93xK9MRh9n2HBf4jQvnwgS8E0l6bMOllXXAyg5eHQFaAgH-t9ayvfj1stxFIDvUENB9bWSH8aq7GFmhYtViUK30iT5PJx8lZ17tw0BRmaA4yXnVsANNe2ezMGNd5vJLnYIhntF0AQAp8xgu39o059eGPVl2KITrulEdm4qs5lJVZ46qO_KvQql6_fHJ_Unqisoca4c_gofK5go8pnyJhDI6z7633LY0swBFWxWmOj-xwYpghDwwkw68_16uXpeyAVoVKMcX0jXrmtAkLm8EQnqgWQpmg';`

(prettier/prettier)